ota-plus-web/app/reactapp-new/package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "./node_modules/webpack/bin/webpack.js -w",
+    "dev": "webpack-dev-server --mode development --open --hot",
```

Comment:
This is not needed in this ticket.

Reply:
its from Lukasz commit, i have made it on top of his change to have server setup done

Reply:
The core project is configures and this is not needed in this case.